### ISSUE TITLE
Could not load file or assembly 'Microsoft.VisualStudio.Composition

### ISSUE THREAD

**Issue by oddi5593** [gitextensions/gitextensions#5990]
Tested with canary builds 
The last worked is GitExtensions-Portable-3.01.00.4558
From 4559 to 4571 crashed on Startup
```
Could not load file or assembly 'Microsoft.VisualStudio.Composition, Version=15.6.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. 
The system cannot find the file specified.
```
If replace files from 4558 👍 
GitUI.dll
GitUI.pdb
GitUIPluginInterfaces.dll
GitUIPluginInterfaces.pdb

starting without problem

- Git Extensions 3.01.00.4571
- Build 03616615c075af006d75e3abed76a4e7f6134ad6 (Dirty)
- Git 2.20.1.windows.1
- Microsoft Windows NT 10.0.17763.0
- .NET Framework 4.7.3260.0
- DPI 96dpi (no scaling)

**Comment by vbjay** [comment]
The lack of details is amazing.  Use our issue template.  It is there for a
reason.

**Comment by RussKie** [comment]
I'm also getting the following trying to run tests:

```
System.IO.FileLoadException : Could not load file or assembly 'Microsoft.VisualStudio.Validation, Version=15.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies. The located assembly's manifest definition does not match the assembly reference. (Exception from HRESULT: 0x80131040)
   at GitUIPluginInterfaces.ManagedExtensibility.SetApplicationDataFolder(String applicationDataFolder)
   at GitUI.PluginRegistry.Initialize() in C:\Development\gitextensions\GitUI\Plugin\PluginRegistry.cs:line 25
   at GitUI.CommandsDialogs.FormBrowse.<<-ctor>b__56_0>d.MoveNext() in C:\Development\gitextensions\GitUI\CommandsDialogs\FormBrowse.cs:line 173
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at GitUI.CommandsDialogs.FormBrowse.<<-ctor>b__56_0>d.MoveNext() in C:\Development\gitextensions\GitUI\CommandsDialogs\FormBrowse.cs:line 179
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at System.Runtime.CompilerServices.TaskAwaiter.HandleNonSuccessAndDebuggerNotification(Task task)
   at System.Runtime.CompilerServices.ConfiguredTaskAwaitable.ConfiguredTaskAwaiter.GetResult()
   at GitUI.ThreadHelper.<>c__DisplayClass13_0.<<FileAndForget>b__0>d.MoveNext() in C:\Development\gitextensions\GitExtUtils\GitUI\ThreadHelper.cs:line 101
--- End of stack trace from previous location where exception was thrown ---
   at System.Runtime.ExceptionServices.ExceptionDispatchInfo.Throw()
   at CommonTestUtils.ConfigureJoinableTaskFactoryAttribute.AfterTest(ITest test) in C:\Development\gitextensions\UnitTests\CommonTestUtils\ConfigureJoinableTaskFactoryAttribute.cs:line 68
   at NUnit.Framework.Internal.Commands.BeforeAndAfterTestCommand.Execute(TestExecutionContext context) in C:\src\nunit\nunit\src\NUnitFramework\framework\Internal\Commands\BeforeAndAfterTestCommand.cs:line 73
   at NUnit.Framework.Internal.Execution.SimpleWorkItem.PerformWork() in C:\src\nunit\nunit\src\NUnitFramework\framework\Internal\Execution\SimpleWorkItem.cs:line 59
```

/cc: @sharwell

**Comment by gerhardol** [comment]
Closed in #6102 for release-3.00 that has been merged to master
